dashboard/public/javascripts/dynamicStyling.js

```diff
@@ -1,135 +1,135 @@
 //random generator for motion
 setInterval(function(){
-    var w=0;
-    for (i=1; i< 6; i++){
-      var x = document.getElementById("motion").rows[i].cells;
-      x[2].innerHTML =  y;
-      if (y<2) {
-        x[2].style.backgroundColor = '#FC6962';
-        w++;
-      }
-      else if (y>90) {
-        x[2].style.backgroundColor = '#C8EEC4';
-      }
-      else {
-        x[2].style.backgroundColor = '#fff';
-      }
+  var w=0;
```

Comment:
Can you write some pseudocode for this file, I'm having a hard time following what is going on.

Reply:
I get a constant repeating error from this file as well
https://i.imgur.com/HRQP1dx.png
Please look into this

Reply:
Fixed the errors, most caused by undefined variables after the random generator was taken out. I put in empty var's right now, but just need to be changed to represent the incoming data. I will add comments to that file as well.